src/utils/py/daos_api.py

```diff
@@ -1930,157 +1960,158 @@ class DaosApiError(Exception):
+    #     CONTAINER.set_attr(data = data)
+    #     print ("=====List Attr =====")
+    #     # commenting out below line, list_attr has no data param
+    #     # size, buf = CONTAINER.list_attr(data = data)
+    #     # print size, buf
+    #     print ("=====Get Attr =====")
+    #     val = CONTAINER.get_attr(data = data)
+    #     for i in range(0, len(data)):
+    #         print(val[i])
 
-        data = {"First":  "1111111111",
-                "Second": "22222222222222222222",
-                "Third":  "333333333333333333333333333333"}
-        print ("=====Set Attr =====")
-        CONTAINER.set_attr(data = data)
-        print ("=====List Attr =====")
-        # commenting out below line, list_attr has no data param
-        # size, buf = CONTAINER.list_attr(data = data)
-        # print size, buf
-        print ("=====Get Attr =====")
-        val = CONTAINER.get_attr(data = data)
-        for i in range(0, len(data)):
-            print(val[i])
+    #     CONTAINER.close()
+    #     print ("container closed {}".format(CONTAINER.get_uuid_str()))
 
-        CONTAINER.close()
-        print ("container closed {}".format(CONTAINER.get_uuid_str()))
+    #     time.sleep(15)
 
-        time.sleep(15)
+    #     CONTAINER.destroy(1)
 
-        CONTAINER.destroy(1)
+    #     print ("container destroyed")
 
-        print ("container destroyed")
+    #     #POOL.disconnect(rubbish)
+    #     #POOL.disconnect()
+    #     #print ("Main past disconnect\n")
 
-        #POOL.disconnect(rubbish)
-        #POOL.disconnect()
-        #print ("Main past disconnect\n")
+    #     #time.sleep(5)
 
-        #time.sleep(5)
+    #     #tgts = [2]
+    #     #POOL.exclude(tgts, rubbish)
+    #     #POOL.exclude_out(tgts, rubbish)
+    #     #POOL.exclude_out(tgts)
+    #     #print ("Main past exclude\n")
 
-        #tgts = [2]
-        #POOL.exclude(tgts, rubbish)
-        #POOL.exclude_out(tgts, rubbish)
-        #POOL.exclude_out(tgts)
-        #print ("Main past exclude\n")
+    #     #POOL.evict(rubbish)
 
-        #POOL.evict(rubbish)
+    #     #time.sleep(5)
 
-        #time.sleep(5)
+    #     #POOL.tgt_add(tgts, rubbish)
 
-        #POOL.tgt_add(tgts, rubbish)
+    #     #print ("Main past tgt_add\n")
 
-        #print ("Main past tgt_add\n")
+    #     #POOL.destroy(1)
+    #     #print ("Pool destroyed")
 
-        #POOL.destroy(1)
-        #print ("Pool destroyed")
+    #     #SERVICE = DaosServer(CONTEXT, b'daos_server', 5)
+    #     #SERVICE.kill(1)
+    #     #print ("server killed!\n")
 
-        #SERVICE = DaosServer(CONTEXT, b'daos_server', 5)
-        #SERVICE.kill(1)
-        #print ("server killed!\n")
-
-    except Exception as EXCEP:
-        print ("Something horrible happened\n")
-        print (traceback.format_exc())
-        print (EXCEP)
-    """
+    # except Exception as EXCEP:
+    #     print ("Something horrible happened\n")
+    #     print (traceback.format_exc())
+    #     print (EXCEP)
+    # """
```

Comment:
Should we remove the beginning and ending `"""` if we are going to comment out this block with `#`s?

Reply:
I seem to be unable to reply to these comments one by one, unfortunately.

Re: list of pylint issues remaining -- yep, am happy to discuss any of these. Some are left untouched based off of previous discussion, ie the "too-many"s all fall under TBD. Judgment calls were made on others, such as the "invalid constant name" warnings. These show up due to the structure of the test file defining all of these vars at module level, as in launch.py, instead of defining them in a function. Making all variables in launch.py CAPS seemed to make it read much worse, so they were left. Perhaps ignores need to be added to these, but the approach has been to not ignore anything explicitly until there's group consensus to ignore.

Re: _iod_ptr, I just tried to keep files as close to intact as possible while fixing issues, didn't want to step much into removing code. I understand that this is doing nothing, though, and can remove.

Re: commented blocks, sure, I can remove them. It does not matter if they are present or not, pylint just does not want a block comment with """s midway through a function or file, those are reserved for docstrings. Block comments midway through functions have to be done via single line #s for pylint to be happy.